devel/google-cloud-sdk/Portfile

```diff
@@ -41,9 +41,25 @@ master_sites        https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/
 
 worksrcdir          ${name}
 
-python.default_version 39
+# Supported Python versions supported according to https://cloud.google.com/sdk/docs/install#mac
+python.versions     35 36 37 38
```

Comment:
this is meaningless in ports that do not start with "py-". Only in those the "python" PG will generate subports and do some other magic. 

Only the "python.default_version" should stay here.

Reply:
Ah yes, I thought this allowed the port to use other versions of Python when already installed, but after reading https://guide.macports.org/chunked/reference.portgroup.html#reference.portgroup.python again that is not what this setting allows. I've removed this now.